File: 4 - Filters/src/TheMallApp/Models/Store.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TheMallApp.Models
{
    public class Store
    {
        public int StoreId { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string WeekDayOpeningHours { get; set; }
        public string WeekEndOpeningHours { get; set; }
        public string Description { get; set; }
        public string StoreUrl { get; set; }
        public string Email { get; set; }
    }
}

File: 2 - Controllers/src/TheMallApp/Scripts/app/domain/Store.ts
module app.domain {
    export interface IStore {
        StoreId: number;
        Name: string;
        Location: string;
        WeekDayOpeningHours: string;
        WeekEndOpeningHours: string;
        Description: string;
        StoreUrl: string;
        Email: string;
    }

    export class Store implements IStore {
        constructor(
            public StoreId: number,
            public Name: string,
            public Location: string,
            public WeekDayOpeningHours: string,
            public WeekEndOpeningHours: string,
            public Description: string,
            public StoreUrl: string,
            public Email: string) {
        }
    }
}